pkg/io/video/convert.go

```diff
@@ -3,18 +3,66 @@ package video
 import (
 	"fmt"
 	"image"
+	"image/color"
 )
 
+// imageToYCbCr converts src to *image.YCbCr and store it to dst
+// Note: conversion can be lossy
+func imageToYCbCr(dst *image.YCbCr, src image.Image) {
+	if dst == nil {
+		panic("dst can't be nil")
+	}
+
+	yuvImg, ok := src.(*image.YCbCr)
+	if ok {
+		*dst = *yuvImg
+		return
+	}
+
+	bounds := src.Bounds()
+	dy := bounds.Max.Y - bounds.Min.Y
+	dx := bounds.Max.X - bounds.Min.X
```

Comment:
Just FYI
```suggestion
	dx := bounds.Dx()
```

Reply:
I didn't know that method existed... Thx